core/src/utils/helpers.ts

```diff
@@ -167,9 +167,14 @@ const ariaAttributes = [
  * Returns an array of aria attributes that should be copied from
  * the shadow host element to a target within the light DOM.
  * @param el The element that the attributes should be copied from.
+ * @param ignoreList The list of aria-attributes to ignore reflecting and removing from the host. Use this in instances where we manually specify aria attributes on the `<Host>` element.
  */
-export const inheritAriaAttributes = (el: HTMLElement) => {
-  return inheritAttributes(el, ariaAttributes);
+export const inheritAriaAttributes = (el: HTMLElement, ignoreList?: string[]) => {
```

Comment:
Did you mean to use this instead of `inheritAttributes` when updating `card` and `item`?

Reply:
Using `inheritAttributes` is intentional. We only want to reflect very specific attributes to the inner `<button>` element inside of `ion-item` and `ion-card`. 

`inheritAriaAttributes` will reflect all aria attributes, including the ones that are manually specified on the `<Host>` element. When I discovered the potential issues with `inheritAriaAttributes`, I posted it in our internal [Slack channel](https://ionicteam.slack.com/archives/C02U5GFFXED/p1664466945191339) about it + added a secondary parameter for future use cases that run into the `<Host>` issue.

Reply:
Can we move this change to a separate PR if it's not needed to resolve the item/card bug?